.github/workflows/deploy.yml

```diff
@@ -146,7 +146,7 @@ jobs:
           path: web/public/
 
   publish:
-    if: github.repository_owner == 'theforeman' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/[0-9]+.[0-9]+'))
+    if: github.repository_owner == 'theforeman'
```

Comment:
Deploy was executed for this PR and it failed. I struggle to find the correct condition. I want the deploy job to be executed if:

* this was not a PR
* owner of the github repo is "theforeman"
* branch is master or X.Y (this can't be done at the moment I will do a workaround for this)

Reply:
> this was not a PR

Looking at https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts and in particular https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context I'd consider looking at `event_name` and check whether it's push.